docs/details.rst

```diff
@@ -44,6 +44,8 @@ The note is printed in the final run of the test in order to include any
 additional information you might need in your test.
 
 
+.. _statistics:
+
 ---------------
 Test Statistics
```

Comment:
Do you need to update the example outputs in this file to reflect the new field?

Reply:
The `test_even_integers` examples in this file both need updating